posts/2023/birth-quick-form/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Birth Quick Form
+date: 2023-05-22
+author: Michael Stenta
+slug: 2023/birth-quick-form
+---
+
+# Birth Quick Form
+
+The farmOS [v2.1.0 release](https://github.com/farmOS/farmOS/releases/tag/2.1.0)
+includes a new and improved *Birth Quick Form* for quickly and easily recording
+animal births. This feature existed in farmOS v1, and has now been ported to v2.
+
+See the video below for a quick demonstration, and read on for more details.
+
+<video autoplay control loop>
```

Comment:
Missing `s` in the control**s** attribute.
Autoplay videos are not the best idea when it comes to accessibility https://www.a11yproject.com/posts/never-use-auto-play/. I would remove the autoplay and loop attributes, so the reader can start the video when they come to the point in the post when it makes sense to watch it and clearly see when it ends.

Reply:
Ah good catch @wotnak! And great insight re: `autoplay` and `loop`. I pushed a fixup: https://github.com/farmOS/farmOS-community-blog/pull/16/commits/8b87f7ea3fa83e7d33ebc337f71a4f5b7d696328